Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/cxf/Saml2ExchangeAuthFrameworkValidatorTest.java

```diff
@@ -130,16 +133,29 @@ public void testValidateSubjectWrongFormat() throws ValidationException {
         when(subject.getNameID()).thenReturn(name);
         when(name.getFormat()).thenReturn("wrong value");
 
-        validator.validate(assertion);
+        validator.validateAssertion(new SamlAssertionWrapper(assertion));
+    }
+
+    /**
+     * Test validate when subject is in X509 format, should not cause exception
+     *
+     * @throws ValidationException the validation exception
+     * @throws WSSecurityException
+     */
+    @Test
+    public void testValidateNameX509validValue() throws WSSecurityException {
+        Saml2ExchangeAuthFrameworkValidator validator = new Saml2ExchangeAuthFrameworkValidator();
+        validator.validateNameIdFormatValue(NhincConstants.AUTH_FRWK_NAME_ID_FORMAT_WINDOWS_NAME, "org/emailAddress");
     }
 
```

Comment:
Added a test to show that it doesn't throw exception.  Do we have a test that shows it will throw exception on error?  Should have both.